<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>264</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="toStartMenuButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>На главную</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutProgramButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="programTab">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>950</width>
      <height>790</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="startTab">
     <attribute name="title">
      <string>start</string>
     </attribute>
     <widget class="QLabel" name="mainWindowLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>940</width>
        <height>281</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="bookTab">
     <attribute name="title">
      <string>book</string>
     </attribute>
     <widget class="QLabel" name="bookLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>940</width>
        <height>717</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QPushButton" name="previousPageButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>740</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPageButton">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>740</y>
        <width>40</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="slashLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>738</y>
        <width>10</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="totalPagesNumberLabel">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>740</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentPageNumberLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>740</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>720</y>
        <width>950</width>
        <height>3</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>false</bold>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="aboutTab">
     <attribute name="title">
      <string>about</string>
     </attribute>
     <widget class="QLabel" name="aboutProgramLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>940</width>
        <height>741</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
